### ISSUE TITLE
top1_acc is allways times 0f 0.05

### ISSUE THREAD

**Issue by Darkalpy** [open-mmlab/mmaction2#1750]
hello, i have trained some models and realized that top1_acc is always times of 0.05 (0.50, 0,55, 0.60 but never 0.57). Is that suppose to be that way or should i change something in my implementation?

**Comment by hukkai** [comment]
@Darkalpy Are you training on your custom dataset or a pubic dataset? And the top1_acc is for train set or validation set?

**Comment by hukkai** [comment]
It it is for train set, it might because your batch size is like 20 and the accuracy is always `correct_samples / 20`.

**Comment by hukkai** [comment]
@Darkalpy If you need further discussion, feel free to re-open the issue. Thanks!
